lib/single-new-eks-opensource-observability-pattern/index.ts

```diff
@@ -52,6 +52,13 @@ export default class SingleNewEksOpenSourceobservabilityPattern {
             );
         }
 
+        if (utils.valueFromContext(scope, "apiserver.pattern.enabled", false)) {
+            ampAddOnProps.enableAPIServerJob = true,
+            ampAddOnProps.ampRules?.ruleFilePaths.push(
+                __dirname + '/../common/resources/amp-config/recording-rules.yml'
```

Comment:
please structure the `'/../common/resources/amp-config/recording-rules.yml'` to `'/../common/resources/amp-config/api-server/recording-rules.yml'`